internal/model/beer_media.go

```
@@ -145,3 +152,11 @@ func (t BeerMediaType) IsLabel() bool {
 func (t BeerMediaType) IsCap() bool {
 	return t == BeerMediaCrownCap || t == BeerMediaTwistOffCap || t == BeerMediaPullOffCap || t == BeerMediaCeramicCap
 }
+
+func RowSizeForPrefix(geoPrefix string) int {
+	smallSheetsGeoPrefixes := []string{"CAUC", "OC", "INDO", "MIDE", "EAAS"}
+	if slices.Contains(smallSheetsGeoPrefixes, geoPrefix) {
+		return 5
```

golangci-lint (2.5.0)

[error] Magic number: 5, in <return> detected

(mnd)